packages/core/src/common/keys.ts

```diff
@@ -57,13 +58,17 @@ export class KeyCode {
      * Parses a string and returns a KeyCode object.
      * @param keybinding String representation of a keybinding
      */
-    public static parse(keybinding: string): KeyCode | undefined {
-        const sequence: string[] = [];
+    public static parse(keybinding: string): KeyCode {
+
+        if (KeyCode.keybindings[keybinding]) {
+            return KeyCode.keybindings[keybinding];
+        }
 
+        const sequence: string[] = [];
         const keys = keybinding.split('+');
```

Comment:
could we trim the segments to allow whitespace, e.g. `ctrlcmd + p`?

Reply:
No since I plan to use space to segment key sequences like:

`ctrl+x f` woulc be ctrl-x followed by f

Reply:
ok